client/client-multi/client-cpp/CMakeLists.txt

```diff
@@ -133,7 +133,7 @@
 #       Default: `0`.
 
 cmake_minimum_required(VERSION 2.8.12 FATAL_ERROR)
-project(KaaCppSdk CXX)
+project(KaaCppSdk C CXX)
```

Comment:
Yep, the `find_package(Threads)` fails if the project does not contain the C language.